src/lib/datepicker/datepicker.spec.ts

```diff
@@ -96,6 +95,20 @@ describe('MdDatepicker', () => {
         expect(document.querySelector('md-dialog-container')).not.toBeNull();
       }));
 
+      it('open in disabled mode should not open the calendar', async(() => {
```

Comment:
It looks like `open` does not require async, so I removed async from other parts of the code as well.